components/logged_in/logged_in.test.tsx

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+import {UserProfile} from 'mattermost-redux/src/types/users';
+import {Channel} from 'mattermost-redux/types/channels';
+
+import LoggedIn from 'components/logged_in/logged_in';
+import BrowserStore from 'stores/browser_store';
+
+jest.mock('actions/websocket_actions.jsx', () => ({
+    initialize: jest.fn(),
+}));
+
+BrowserStore.signalLogin = jest.fn();
+
+type Props = {
```

Comment:
Instead of repeating this type definition, you can export it from `logged_in.tsx` and re-import it here